js/alumni.js

```diff
@@ -372,6 +374,8 @@ function initMap() {
                 '</a>' +
                 "</div>";
             infoWindow.setContent(contentString);
+            map.setZoom(5);
+            map.setZoom(7);
```

Comment:
info: what is this doing exactly?

Reply:
when focusing it zooms, there's a bit of a zoom-out then in, but it's not really noticeable